datashader/tests/test_dask.py

```diff
@@ -89,10 +102,28 @@ def _dask_cudf_DataFrame(*args, **kwargs):
     )
     return dask_cudf.from_cudf(cdf, npartitions=2)
 
+
+@dask_switcher(query=True, extras=["dask_cudf"])
+def _dask_expr_cudf_DataFrame(*args, **kwargs):
+    import cudf
+    import dask_cudf
+
+    if Version(dask_cudf.__version__) >= Version("24.06"):
+        pytest.skip("dask-expression first requires dask-cudf 24.06")
+
+    if kwargs.pop("geo", False):
+        pytest.skip("dask-cudf currently does not work with spatialpandas")
```

Comment:
Can you specify versions so that we can tell how current this information is, like "As of dask-cudf version 1.3, dask-cudf is not compatible with spatialpandas version 0.6".